.github/workflows/tmp-codecove.yaml

```diff
@@ -27,6 +27,7 @@ concurrency:
   cancel-in-progress: true
 jobs:
   coverage:
+    timeout-minutes: 680
```

Comment:
We should reduce this for cost reasons and DX. Understand this is just for testing, just don't want to have a workflow running for long and end up costing us some.